src/query/shorthand.ts

```diff
@@ -91,6 +100,54 @@ export function spec(specQ: SpecQuery,
   return parts.join('|');
 }
 
+let getFilterExpression = filter.getFilterExpression;
```

Comment:
If you have done this https://github.com/vega/vega-lite/issues/1491, you won't have this problem.